README.md

```diff
@@ -159,10 +159,10 @@ You'll need a version with OpenSSL version 1.0.0 or later.
 
 ### Hostname doesn't match
 
-If you get an `ssl.CertificateError` complaining about hostname match, your installed packages does not meet version [requirements](requirements.txt).
-Specifically check `ipaddress` and `urllib3` package versions to make sure they met requirements in [requirements.txt](requirements.txt) file.
+If you get an `ssl.CertificateError` complaining about hostname match, your installed packages do not meet version [requirements](requirements.txt).
+Specifically, check `ipaddress` and `urllib3` package versions to make sure they met requirements in [requirements.txt](requirements.txt) file.
```

Comment:
met -> meet

Reply:
revision done. waiting for further feedback.

Reply:
it's in make sure they met requirements -> make sure they meet requirements